tox.ini

```diff
@@ -100,8 +100,7 @@ commands =
 [testenv:integration]
 description = Run integration tests
 deps =
-    # Last compatible version with Juju 2.9
-    juju==3.0.4
+    juju==3.6.*
```

Comment:
Is this intended?

Reply:
Yes, otherwise I get https://github.com/canonical/is-charms-template-repo/actions/runs/12648296234/job/35242642410?pr=22#step:27:146, I think its the ws socket issue we saw in the past which has been fixed in recent releases. I don't know why we need juju 2.9 for new repos, I think it's ok to start new charms fresh with juju 3.6.

Reply:
I think maybe we can update the `juju-channel` in the template to `3/stable` as that's the new stable version.

https://github.com/canonical/is-charms-template-repo/blob/822c22c68826e584b9fdbb86b03e077434684ef3/.github/workflows/integration_test.yaml#L10